From b1f011c77ea0167674728803d31b903c3da5ae51 Mon Sep 17 00:00:00 2001
From: Rinigus <rinigus.git@gmail.com>
Date: Sat, 29 Apr 2023 18:23:40 +0300
Subject: [PATCH] Disable X11

---
 src/runtime/kwalletd/kwalletd.cpp | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/runtime/kwalletd/kwalletd.cpp b/src/runtime/kwalletd/kwalletd.cpp
index 5914be9..bdbe172 100644
--- a/src/runtime/kwalletd/kwalletd.cpp
+++ b/src/runtime/kwalletd/kwalletd.cpp
@@ -35,12 +35,12 @@
 #include <kwalletentry.h>
 #include <kwindowsystem.h>
 
-#if !defined(Q_OS_WIN) && !defined(Q_OS_MAC)
-#define HAVE_X11 1
-#include <KX11Extras>
-#else
+// #if !defined(Q_OS_WIN) && !defined(Q_OS_MAC)
+// #define HAVE_X11 1
+// #include <KX11Extras>
+// #else
 #define HAVE_X11 0
-#endif
+// #endif
 
 #ifdef HAVE_GPGMEPP
 #include <gpgme++/key.h>
-- 
2.39.2

